criteria/document-maturity.md

```diff
@@ -11,13 +11,15 @@ redirect_from:
 ## Requirements
 
 * A codebase MUST be versioned.
-* A codebase that is ready to use MUST only depend on other codebases that are also ready to use.
-* A codebase that is not yet ready to use MUST have one of the labels: prototype, alpha, beta or pre-release version.
+* A codebase version that is ready to use MUST only depend on other codebase versions that are also ready to use.
+* A codebase which has no versions ready for use MUST document that clearly.
+* A codebase version that is not yet ready to use MUST use semantic versioning with a 0 for the major version number or have one of the labels: pre-alpha, alpha, beta or release candidate.
```

Comment:
Is this “label” in the sense of [SemVer§9](https://semver.org/#spec-item-9) or independent from the release / packaging system? The current phrasing is ambiguous and I initially understood it as SemVer§9 until I read label `release candidate` with a space, which would not be a valid SemVer label.

Reply:
Today, this is not [SemVer§9](https://semver.org/#spec-item-9). One can name the release separate from the semver tag, consider https://github.com/publiccodenet/standard/releases/ which so far have labels of the form "tenth update" and version tag of `0.3.0`.

That said, it would clearly be much, much better if they could be exactly the same. Thus, 'release candidate' creates a problem. This should absolutely be addressed before merging.